### ISSUE TITLE
seeding from template

### ISSUE THREAD

**Issue by majewsky** [majewsky/portunus#3]
There should be a way to seed technical users from config. Something like

```shellSession
$ cat seed.json
{
  "users": [ {
     "name": "foo",
      "passwordFromCommand": "cat /etc/secrets/foo-password.txt",
      "groups": [ "bar" ],
      ...
   } ],
   "groups": [ ...]
}
```

that gets merged with `database.json` on startup.

**Comment by SuperSandro2000** [comment]
This would be really convenient for the Nixos module because merging json lists requires extra handling and having manual steps to setup a module is not really great.

Substituting a placeholder password string with the real secret isn't to hard but it would be convenient if an environment variable could be used instead which can be easily loaded from sops-nix with the help of systemd's EnvironmentFile setting.

**Comment by majewsky** [comment]
This is implemented in v1.1.0-beta.1.
